addons/udes_priorities/models/priority.py

```diff
@@ -1,15 +1,17 @@
 # -*- coding: utf-8 -*-
 
-from odoo import models, fields, api, _
+from odoo import models, fields, api, SUPERUSER_ID, _
 from odoo.exceptions import UserError
 from uuid import uuid4
 
+
+MAIN_PROTECTED_FIELDS = {"reference"}
+
 PROTECTED_FIELDS = {
     "name",
-    "reference",
     "sequence",
     "picking_type_ids",
-    "group_ids",
+    "priority_group_ids",
     "active",
```

Comment:
I think we should only have `name` and `picking_type_ids` in `PROTECTED_FIELDS` and the rest should be in `MAIN_PROTECTED_FIELDS`

Reply:
Yeah good point, if we make a priority inactive I can see that breaking a lot of things... but if it is protected then why even have an active field?

Maybe we need a constraint on active field, that it can't be set inactive if it is used on other pickings. Otherwise we'll risk getting an error when accessing a picking that is using an inactive priority.

Reply:
Done.